zipline/_protocol.pyx

```diff
@@ -422,12 +496,17 @@ cdef class SidView:
             column = "close"
         elif column == "open_price":
             column = "open"
+        elif column == "dt":
+            return self.dt
```

Comment:
These seem unnecessary, given that `__getattr__` is called when no instance or class attribute was found.